### ISSUE TITLE
background-grid: right margin too wide when using split gutters

### ISSUE THREAD

**Issue by marcvangend** [at-import/Singularity#130]
I'm testing a grid with the following settings:

```
$grids: 3;
$gutters: .1;
$gutter-styles: 'split';
```

Positioning elements on this grid works as expected, but when I enable the background-grid, the gutter on the right edge of the grid is twice as wide as it should be. The generated CSS is:

```
  background-image: linear-gradient(left, rgba(210, 105, 30, 0.25), rgba(210, 105, 30, 0.25) 1.51515%, rgba(210, 105, 30, 0.5) 1.51515%, rgba(210, 105, 30, 0.5) 30.30303%, rgba(210, 105, 30, 0.25) 30.30303%, rgba(210, 105, 30, 0.25) 33.33333%, rgba(210, 105, 30, 0.5) 33.33333%, rgba(210, 105, 30, 0.5) 63.63636%, rgba(210, 105, 30, 0.25) 63.63636%, rgba(210, 105, 30, 0.25) 66.66667%, rgba(210, 105, 30, 0.5) 66.66667%, rgba(210, 105, 30, 0.5) 96.9697%, rgba(210, 105, 30, 0.25) 96.9697%, rgba(210, 105, 30, 0.25) 98.48485%);
```

**Comment by Snugug** [comment]
If you are in Google Chrome, please check in Firefox as [Chrome does not render gradients well](https://github.com/Team-Sass/Singularity/wiki/Creating-Grids#grid-backgrounds-may-be-inaccurate)

**Comment by marcvangend** [comment]
Thanks for the tip. Unfortunately I see the same result in Chromium (version 29) and Firefox (version 23) on Linux.

**Comment by ducdebreme** [comment]
I experienced the same issue. (See image)
![selection_014](https://f.cloud.github.com/assets/2411302/1619992/e23d89a8-5668-11e3-9bca-3055a51d9cba.png)

**Comment by Snugug** [comment]
This was all sorts of wrong before. Resolved in #157  (which will become v1.2.0 of Singularity)
